lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -6,17 +6,22 @@
  */
 @use '../../scss-shared/nx-focus-border-helpers';
 
-$-height: 16px;
-$-width: 16px;
+$-size: 16px;
 
 .nx-radio__circle {
-  height: $-height;
-  width: $-width;
+  height: $-size;
+  width: $-size;
+
+  border-radius: $-size;
+}
+
+.nx-radio__outer-circle {
+  stroke-width: var(--nx-radio-checkbox-border-width);
```

Comment:
> This isn't working correctly. If you remove the `border-radius` you can see that the `nx-radio__outer-circle` is actually thicker than what's visible with that border radius, and gets cut off. Conversely, if you add `display: none` to `.nx-radio__outer-circle` you can see the visible border increase towards the inside. Overall I think there's a real problem in terms of how the outer circle is drawn: it's effectively drawn by _both_ the box-shadow and the svg `<circle>`, with each of those two drawing slightly different things neither of which are quite right.
> 
> Part of the problem is that the `r` on the `.nx-radio__outer-circle` needs to be decreased by 0.5 in dark mode to accommodate the increased `stroke-width`. That could be challenging as that property only became CSS-controllable in SVG 2 which has unclear browser support. So the best path forward may depend on what you find in terms of browser support for that.
> 
> I think a bigger re-think is needed here. Some options I can suggest:
> 
> * introduce a wrapper element around the `nx-radio-checkbox__control` that just establishes the positioning context for the focus border, but which doesn't draw anything itself.  Then the borders on the `nx-radio-checkbox__control` can go back to being drawn the way they used to be.
> * Switch to `svg` for `NxCheckbox` so that it matches `NxRadio` in that regard, using a `<path>` to draw its border.  As I write this I realize it's probably the harder option.
> * **If the radius of the `nx-radio__outer-circle` cannot be controlled via CSS** then you'll probably need to stick with something more like your current `box-shadow` solution, but getting rid of `nx-radio__outer-circle` as it's essentially just in the way right now.

Thanks, did a refactor here: c48a8acbcfd31f99314d410d180ece3c046a6577
I still need to update visual tests and add comment.